src/plugins/discover/public/application/components/loading_spinner/loading_spinner.test.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+import { ReactWrapper } from 'enzyme';
+import { LoadingSpinner } from './loading_spinner';
+// @ts-ignore
+import { findTestSubject } from '@elastic/eui/lib/test';
+
+describe('loading spinner', function() {
+  let component: ReactWrapper;
+
+  it('LoadingSpinner renders a Searching text', () => {
```

Comment:
I think these 2 tests could be replaced with one as they are quite simple. Either with:
```
it('LoadingSpinner renders a "searching" text and a spinner', () => {
    component = mountWithIntl(<LoadingSpinner />);
    expect(findTestSubject(component, 'loadingSpinnerText').text()).toBe('Searching');
    expect(findTestSubject(component, 'loadingSpinner').length).toBe(1);
  });
```

or with a snapshot:
```
  component = mountWithIntl(<LoadingSpinner />);
  expect(component).toMatchSnapshot();
```

I haven't run the code.

Reply:
I think it is better to avoid the snapshots here but yes they could be combined.